x-pack/plugins/observability_solution/apm_data_access/kibana.jsonc

```diff
@@ -8,7 +8,7 @@
     "browser": false,
     "configPath": ["xpack", "apm_data_access"],
     "requiredPlugins": ["data"],
-    "optionalPlugins": [],
+    "optionalPlugins": ["security"],
```

Comment:
tested in serverless and the plugin works fine

Reply:
Q: Is this needed for serverless only?

Reply:
Just wanted to make sure everything worked on serverless. Some plugins are disabled for oblt project.

Reply:
I see thanks for explaining, I didn't check serverless but it was fine running it locally (not in serverless) 👍 I was thinking that it is optional because of serverless but it makes sense to be also for non-serverless.